inc/Engine/Preload/Links/Subscriber.php

```diff
@@ -122,13 +122,13 @@ public function add_preload_script() {
 	 */
 	private function get_preload_links_config() {
 		$use_trailing_slash = $this->use_trailing_slash();
-		$images_ext         = 'jpg|jpeg|gif|png|tiff|bmp|webp|avif';
+		$images_ext         = 'jpg|jpeg|gif|png|tiff|bmp|webp|avif|pdf|doc|docx|xls|xlsx|php';
 
 		$config = [
 			'excludeUris'       => $this->get_uris_to_exclude( $use_trailing_slash ),
 			'usesTrailingSlash' => $use_trailing_slash,
 			'imageExt'          => $images_ext,
-			'fileExt'           => $images_ext . '|php|pdf|html|htm',
+			'fileExt'           => $images_ext . '|php|html|htm',
```

Comment:
Thanks for the PR @jeawhanlee. I think `php` here is duplicated now. what do you think?

Reply:
@Mai-Saad You are right!